test/e2e/helpers/user-storage/userStorageMockttpController.ts

```diff
@@ -0,0 +1,196 @@
+import { CompletedRequest, Mockttp } from 'mockttp';
+
+// TODO: Export user storage schema from @metamask/profile-sync-controller
```

Comment:
Agreed as discussed, we can export to shared once we are satisfied with this pattern for mobile :)